actor/src/main/java/org/apache/pekko/dispatch/AbstractBoundedNodeQueue.java

```diff
@@ -11,6 +11,29 @@
  * Copyright (C) 2009-2022 Lightbend Inc. <https://www.lightbend.com>
  */
 
+/*
```

Comment:
I think the existing comment far more clearly covers what happened. There was no code copied from any of both projects but rather ideas. If anything, the code from Andriy Plokhotnuyk has methods that are somewhat similar.

Reply:
@justinmclean do you have suggestions for this? Should we treat ideas separately from source?

Reply:
IANAL but usually there's no copyright on the ideas themselves. The question is if enough code has been copied or literally translated to warrant a copyright header. I'd say, it isn't enough code and the existing comment faithfully represents that the code has been *based* on that code and ideas without being a literal copy. As mentioned in https://github.com/apache/incubator-pekko/issues/425#issuecomment-1598371026, we can add a small notice to the LICENSE file to clarify that (whether or not we might need a license), give credit and specify that the existing licenses were permissive.